Dockerfile-legacy

```diff
@@ -128,7 +127,7 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
         mercurial \
         subversion \
         # Install package managers (in versions known to work).
-        cargo=$CARGO_VERSION \
+        cargo \
```

Comment:
Please remove the `Nicolas Nobelis <nicolas.nobelis@bosch.com>` line (above the signoff) from the commit message.